internal/load-test/query_load.py

```diff
@@ -1,15 +1,17 @@
-import argparse, sys, logging
+import argparse, sys, os, logging
 from core.keys import keys_show
 from modules.auth.query import account_type
 from modules.bank.query import query_balances
 from modules.staking.query import query_staking_delegations, query_staking_validators
 from utils import validate_num_txs
 from stats import print_stats, clear_data_by_type, QUERY_TYPE
 
+num_txs = int(os.getenv("NUM_TXS"))
```

Comment:
can we convert it to something like `num_txs = utils.env.NUM_TXS` where the package will have 
```
NUM_TXS = int(os.getenv("NUM_TXS", 50))
```